js/level3.js

```diff
@@ -261,7 +261,7 @@
 
     Example:
 
-    var pageNode = document.querySelector('body')[0];
```

Comment:
Accessing `[0]` on a single Element is going to give you an error. You only need to do `[0]` if you're using `document.querySelectorAll`